.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,54 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
```

Comment:
do we need to match version numbers here?

Reply:
Seems like it only exist master so I remvoed the version numbers